queries/bash/highlights.scm

```diff
@@ -103,6 +106,9 @@
 
 (variable_name) @variable
 
+((variable_name) @constant
+ (#match? @constant "^[A-Z][A-Z_0-9]*$"))
```

Comment:
+ instead of * ?

Reply:
The first character class already covers one var character.

Reply:
if that's what you mean

Reply:
stupid me.